Tests/SwiftParserTest/translated/InitDeinitTests.swift

```diff
@@ -121,9 +151,12 @@ final class InitDeinitTests: XCTestCase {
       }
       """,
       diagnostics: [
-        // TODO: Old parser expected error on line 2: no parameter clause allowed on deinitializer, Fix-It replacements: 9 - 18 = ''
-        DiagnosticSpec(message: "unexpected code '(a : Int) {}' in class"),
-      ]
+        DiagnosticSpec(message: "deinitializers cannot have parameters", fixIts: ["remove function signature"]),
+      ], fixedSource: """
+      class FooClassDeinitializerA {
+        deinit{}
```

Comment:
Unfortunate that we remove the trailing trivia here. But adding it in would make the `deinit x(a: Int) {` case look silly instead 🤷.